### ISSUE TITLE
[Feature Request] `--extra_liners` option

### ISSUE THREAD

**Issue by shufo** [shufo/blade-formatter#842]
**Ref**:

```
  -E, --extra_liners                 List of tags (defaults to [head,body,/html] that should have an extra newline before them.
```

---

### Source (index.html)

```html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
  </head>
  <body>
    <p>Dummy</p>
  </body>
</html>
```

### -I option

> npx js-beautify -I index.html

```html
<!DOCTYPE html>
<html lang="en">

    <head>
        <meta charset="UTF-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <title>Document</title>
    </head>

    <body>
        <p>Dummy</p>
    </body>

</html>
```

### [Feature Request] add -E option

> npx js-beautify -I -E= index.html

```html
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <title>Document</title>
    </head>
    <body>
        <p>Dummy</p>
    </body>
</html>
```

_Originally posted by @yaegassy in https://github.com/shufo/blade-formatter/issues/840#issuecomment-1650634111_

**Comment by ldommer** [comment]
@shufo

I think I wrote an email to you some months ago, asking for this feature.

Today I reached out for your plugin again and made it a file watcher in my PhpStorm settings.

I remembered missing this feature, forked your repo and added this functionality as an option for your Balde Formatter, modifying the behaviour of [html-beatify](https://github.com/beautify-web/js-beautify#css--html).

I already pushed my feature branch and try to link it to this issue.
